modules/cucumber/modules/core/src/main/java/com/smartbear/readyapi/readyapi4j/cucumber/builders/Assertions.java

```diff
@@ -0,0 +1,135 @@
+package com.smartbear.readyapi.readyapi4j.cucumber.builders;
+
+import com.smartbear.readyapi.client.model.GroovyScriptAssertion;
+import com.smartbear.readyapi.client.model.JsonPathContentAssertion;
+import com.smartbear.readyapi.client.model.RequestTestStepBase;
+import com.smartbear.readyapi.client.model.ResponseSLAAssertion;
+import com.smartbear.readyapi.client.model.SimpleContainsAssertion;
+import com.smartbear.readyapi.client.model.ValidHttpStatusCodesAssertion;
+import com.smartbear.readyapi.client.model.XPathContainsAssertion;
+
+import java.util.List;
+
+/**
+ * Utility class with static method for building various types of assertions
+ */
+
+public class Assertions {
```

Comment:
Some of the methods in this class are identical to methods in Assertions class in readyapi4j-core module. I think either we can remove the duplicates or delegate to Assertions class in readyapi4j-core for duplicates.